pkg/controller/logcollector/logcollector_controller.go

```diff
@@ -245,6 +247,67 @@ func (r *ReconcileLogCollector) Reconcile(request reconcile.Request) (reconcile.
 		}
 	}
 
+	if instance.Spec.AdditionalStores != nil {
+		if instance.Spec.AdditionalStores.Syslog != nil {
+			syslog := instance.Spec.AdditionalStores.Syslog
+
+			// Try to grab the ManagementClusterConnection CR (if present) because we need it for some
+			// validation with respect to Syslog.logTypes.
+			managementClusterConnection, mccErr := utils.GetManagementClusterConnection(ctx, r.client)
```

Comment:
Should check if mccErr is IsNotFound and if anything else then we should degrade on that err.

Reply:
Might even want to explicitly set managmenetClusterConnection = nil in the case of IsNotFound.